persistence/build.gradle

```diff
@@ -0,0 +1,3 @@
+dependencies {
+    compile project(':core-models')
```

Comment:
```suggestion
    api project(':core-models')
```

Reply:
Neat, I did not know that Gradle could do this.